dojo/forms.py

```diff
@@ -519,6 +519,31 @@ def clean_scan_date(self):
         return date
 
 
+class ImportEndpointMetaForm(forms.Form):
+    file = forms.FileField(widget=forms.widgets.FileInput(
+        attrs={"accept": ".csv"}),
+        label="Choose meta file",
+        required=True)  # Could not get required=True to actually accept the file as present
+    create_endpoints = forms.BooleanField(
+        label="Create Nonexsitng Endpoint",
+        initial=True,
+        required=False,
+        help_text="Create hosts in meta that do no already exist",)
+    create_tags = forms.BooleanField(
+        label="Add Tags",
+        initial=True,
+        required=False,
+        help_text="Add meta from file as tags in the format key:value",)
+    create_dojo_meta = forms.BooleanField(
+        label="Add DojoMeta",
+        initial=False,
+        required=False,
+        help_text="Add meta from file as DojoMeta. DojoMeta is primarily used for displaying custom fields.",)
```

Comment:
`DojoMeta` is the name of the class, but it's not used in the user interface. So users will not know what this term means.

```suggestion
        help_text="Add data from file as Metadata. Metadata is used for displaying custom fields.",)
```

Reply:
Sure